src/templates/header/partials/nav-alternate-languages.hbs

```diff
@@ -0,0 +1,12 @@
+{{#if isMultiLingual}}
+    {{#each alternativeLanguages as | lang |}}
+        <li class="c-nav-list-item u-showAboveMid" data-test-id="languageToggle">
+            <a href="{{ lang.returnUrl }}" class="c-nav-list-link">
+                <span class="is-visuallyHidden">
+                    {{#with ../this}}{{ i18n "languageSwitcherAccessibility" language=lang.name }}{{/with}}
```

Comment:
This is an unfortunate feature (in this context) of handlebars which means that the scope of `this` changes inside an `each` loop so we have to tell it to look *up* one scope level with `../this` so that it can access the `i18n` helper.

It's only a minor inconvenience, but would be nice if we could it in a better way if possible in the future.